ui-tests/cypress/support/restAPI/emailConfigAPI.js

```diff
@@ -0,0 +1,43 @@
+const apiURL     = Cypress.config('restAPI');
+
+const controller = `${apiURL}plugins/emailSettings/SMTPServer`;
+
+Cypress.Commands.add('emailConfigController', () => {
+  cy.get('@tokens').then(tokens => {
+    return new EmailConfigController(tokens.access_token);
+  });
+});
+
+class EmailConfigController{
+    
+    constructor(access_token) {
+        this.access_token = access_token;
+      }
+
+      options = {  
```

Comment:
is this a constant? which options does it describe?

Reply:
yes, it's a constant.  "smtpDefaultSettings" I think is better.